go-controller/pkg/node/udn_isolation.go

```diff
@@ -491,6 +491,10 @@ func (m *UDNHostIsolationManager) getPodInfo(podKey string, pod *v1.Pod) (*podIn
 	// only add pods with primary UDN
 	primaryUDN, err := m.isPodPrimaryUDN(pod)
 	if err != nil {
+		if util.IsAnnotationNotSetError(err) {
```

Comment:
Just a heads that (unfortunately) this won't be a `IsAnnotationNotSetError` if the annotation is set but just not for a specific network.

Reply:
yeah that part I expected, but I always ask for "default" network, so this shouldn't be a problem